app/src/main/java/org/p2p/wallet/user/repository/UserLocalRepository.kt

```diff
@@ -7,6 +7,7 @@ import org.p2p.wallet.receive.list.TokenListData
 
 interface UserLocalRepository {
     fun setTokenPrices(prices: List<TokenPrice>)
+    fun getTokenPrices(): Flow<List<TokenPrice>>
```

Comment:
isFlowNeeded here? Maybe return just List<>?

Reply:
I think all layers above presenter better to return Flow or use suspend. Flow for reactive, suspend for async.
https://developer.android.com/kotlin/coroutines/coroutines-best-practices#coroutines-data-layer